### DISCUSSION TITLE
Full German translations

### DISCUSSION THREAD

**Discussion by eawmaas** [xenocrat/chyrp-lite#216] [Show and tell]
As promised, in the attachments the German translations for Chyrp Lite including all standard components, the file-structure is as it should be, the pot-files are in there too and still original ofcourse.

I have been testing around a bit and it looks to be working fine.
[locales_de_DE.zip](https://github.com/xenocrat/chyrp-lite/files/12909214/locales_de_DE.zip)

**Comment by xenocrat** [comment]
Another fantastic contribution - thank you! I'll get these integrated for the next release.

**Reply by eawmaas** [reply]
You're very welcome! Looking forward to the next release.
